python/srv.py

```diff
@@ -25,23 +25,34 @@
 
 logging.basicConfig(level=logging.INFO)
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    'database', help='path to database file, in-memory will be used if empty')
+parser.add_argument(
+    'url', help='either UNIX:<socket_path> or <ip>:<tcp port> to listen on')
+parser.add_argument(
+    'auth_key', help='hex-encoded up to 64bytes value that '
+                     'clients need to provide when connecting')
+parser.add_argument('plugin', nargs='*', help='name plugin module to load')
```

Comment:
as `plugin` is optional, we can  change it to `--plugin` so it will not clash with database. Use `nargs=1, action='append'`